charts/hail-search/templates/persistentvolume.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.persistentVolume.enabled -}}
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: {{ include "hail-search.fullname" . }}-pv
+  labels:
+    {{- include "hail-search.labels" . | nindent 4 }}
+spec:
+  storageClassName: ""
```

Comment:
i'm not actually sure what happens with an empty class name specified here (i think GKE has 2 or 3 classes available by default. This could also not matter/be moot if you're just referencing persistent disks that already exist in GCE.

Reply:
this was recommended/documented somewhere in the gke docs and [here](https://stackoverflow.com/questions/58622284/can-i-combine-storageclass-with-persistentvolume-in-gke)... I think it's moot because the persistent disks already exist and we're not asking for dynamic provisioning

the storage classes do exist already by virtue of gke it appears:

```
bblanken@wmbbe-67c seqr-pipeline-airflow % kubectl get storageclass | grep pd
E0329 17:03:55.507374   11981 memcache.go:255] couldn't get resource list for external.metrics.k8s.io/v1beta1: Got empty response for: external.metrics.k8s.io/v1beta1
airflow-redis-persistence-sc   pd.csi.storage.gke.io          Delete          Immediate              true                   9d
premium-rwo                    pd.csi.storage.gke.io          Delete          WaitForFirstConsumer   true                   9d
standard                       kubernetes.io/gce-pd           Delete          Immediate              true                   9d
standard-rwo (default)         pd.csi.storage.gke.io          Delete          WaitForFirstConsumer   true                   9d
```

potentially worth re-evaluating if we're gonna support this for open-source though.